src/goto-programs/adjust_float_expressions.cpp

```diff
@@ -69,9 +69,11 @@ static bool have_to_adjust_float_expressions(const exprt &expr)
       return true;
   }
 
-  forall_operands(it, expr)
-    if(have_to_adjust_float_expressions(*it))
+  for(const auto &op : expr.operands())
+  {
```

Comment:
I like the brackets here but I don't know if it is compliant with the style guide...

Reply:
CODING_STANDARD.md says "put braces around multi-line blocks," so I think this is desired.

Reply:
Thanks.